subsys/bluetooth/shell/bt.c

```diff
@@ -1276,6 +1292,8 @@ static int cmd_init(const struct shell *sh, size_t argc, char *argv[])
 		}
 	}
 
+	k_work_init_delayable(&active_scan_timeout_work, active_scan_timeout);
+
```

Comment:
You can actually defined the `k_work` as 

```c
static K_WORK_DELAYABLE_DEFINE(active_scan_timeout_work, active_scan_timeout);
```

Then you both declare and initialize it in one go

Reply:
OK.